### PR TITLE
[SDP-1258] Add function to migrate data from single tenant to multi-tenant instance

### PR DESCRIPTION
**Pull request by marwen-abid**
### What
Function to migrate single tenant to multi-tenant instance. 

Can run the migration with the following statement : 
```sql
SELECT admin.import_tenant_data_from_v1_to_v2('unhcr');
```

### Why
Facilitating data migration for users. 

### Known limitations

[TODO or N/A]

#### PR Structure

* [ ] This PR has a reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR title and description are clear enough for anyone to review it.
* [ ] This PR does not mix refactoring changes with feature changes (split into two PRs otherwise).

#### Thoroughness

* [ ] This PR adds tests for the new functionality or fixes.
* [ ] This PR contains the link to the Jira ticket it addresses.

#### Configs and Secrets

* [ ] No new CONFIG variables are required -OR- the new required ones were added to the helmchart's [`values.yaml`] file.
* [ ] No new CONFIG variables are required -OR- the new required ones were added to the deployments ([`pr-preview`], [`dev`], [`demo`], `prd`).
* [ ] No new SECRETS variables are required -OR- the new required ones were mentioned in the helmchart's [`values.yaml`] file.
* [ ] No new SECRETS variables are required -OR- the new required ones were added to the deployments ([`pr-preview secrets`], [`dev secrets`], [`demo secrets`], `prd secrets`).

#### Release

* [ ] This is not a breaking change.
* [ ] **This is ready for production.**. If your PR is not ready for production, please consider opening additional complementary PRs using this one as the base. Only merge this into `develop` or `main` after it's ready for production!

#### Deployment

* [ ] Does the deployment work after merging?

[`values.yaml`]: ../helmchart/sdp/values.yaml
[`pr-preview`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/common-previews/stellar-disbursement-platform/backend-helm-values
[`dev`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/backend-helm-values
[`demo`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/demo/demo-backend-helm-values
[`pr-preview secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/common-previews/externalsecrets-common-previews.yaml#L241-L346
[`dev secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/stellar-disbursement-platform-externalsecrets.yaml
[`demo secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/demo/demo-sdp-externalsecrets.yaml

### CODE REVIEW

File: db/migrations/admin-migrations/2024-07-15.0-migrate-data-v1-to-v2-function.sql [hunk 1]
```diff
@@ -0,0 +1,99 @@
+-- +migrate Up
```
Comment (marwen-abid) [inline db/migrations/admin-migrations/2024-07-15.0-migrate-data-v1-to-v2-function.sql:1]: Original import script https://developers.stellar.org/platforms/stellar-disbursement-platform/admin-guide/single-tenant-to-multi-tenant-migration#importing-your-data

File: db/migrations/admin-migrations/2024-07-15.0-migrate-data-v1-to-v2-function.sql [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+-- +migrate StatementBegin
+CREATE OR REPLACE FUNCTION import_tenant_data_from_v1_to_v2(tenant_name TEXT) RETURNS void AS $$
```
Comment (marcelosalloum) [inline db/migrations/admin-migrations/2024-07-15.0-migrate-data-v1-to-v2-function.sql:4]: WDYT about renaming it from `import...` to `migrate...`?

File: db/migrations/admin-migrations/2024-07-15.0-migrate-data-v1-to-v2-function.sql
Comment (marcelosalloum) [inline db/migrations/admin-migrations/2024-07-15.0-migrate-data-v1-to-v2-function.sql:1]: Also, this sounds intuitive to me but can we add a comment to this file stating how this command is meant o be used?

**Review by marcelosalloum — APPROVED** [review]
LGTM! Plz, feel free to merge it after addressing the comments below.

File: db/migrations/admin-migrations/2024-07-15.0-migrate-data-v1-to-v2-function.sql (hunk 2 above)
Comment (marwen-abid) [reply db/migrations/admin-migrations/2024-07-15.0-migrate-data-v1-to-v2-function.sql]: Addressed in b405615acc94fe7a87822fa65

File: db/migrations/admin-migrations/2024-07-15.0-migrate-data-v1-to-v2-function.sql
Comment (marwen-abid) [reply db/migrations/admin-migrations/2024-07-15.0-migrate-data-v1-to-v2-function.sql:1]: Addressed in b405615acc94fe7a87822fa65

**Pull request merged**